src/requisitionMerge.js

```diff
@@ -163,19 +163,23 @@ function requisitionItemsMerge(incomingRequisitionLines, outgoingRequisitionLine
  * @return {Object} the updated eSIGL requisition with incoming values applied.
  */
 export default function requisitionMerge(incomingRequisition, outgoingRequisition) {
-  const { requisitionLines: incomingLines } = incomingRequisition;
+  const { requisitionLines: incomingLines, custom_data } = incomingRequisition;
   const { fullSupplyLineItems: outgoingLines, regimenLineItems } = outgoingRequisition;
+  const regimenData = custom_data.regimenData || [];
   // If there are no outgoing lines, nothing will be pushed. Throw an error here as
   // something has gone wrong and no requisitions for this program and facility tuple
   // will be pushed until it is fixed.
+
   if (!outgoingLines || !outgoingLines.length) throw errorObject(ERROR_MERGE_PARAMS, 'outgoing');
-  // Regimen items are not pushed into eSIGL. They are required for validation, Set each lines
-  // required fields for validation to 0.
+  // Regimen items are required for validation. Set the value of each item, defaulting to 0.
   if (regimenLineItems) {
     regimenLineItems.forEach(regimenItem => {
-      regimenItem.patientsOnTreatment = 0;
+      const matchingItem = regimenData.find(regimenDatum => regimenDatum.code === regimenItem.code);
+      const value = matchingItem === null ? 0 : matchingItem.value || 0;
+      regimenItem.patientsOnTreatment = value;
```

Comment:
I think this works for the simple case of there being one column:

![image](https://user-images.githubusercontent.com/35858975/69205721-a3973700-0baf-11ea-99fa-bcf132be8791.png)
![image](https://user-images.githubusercontent.com/35858975/69205735-b27de980-0baf-11ea-9e77-115b187c7ab6.png)

But not for the more complex cases with multi column
![image](https://user-images.githubusercontent.com/35858975/69205771-cfb2b800-0baf-11ea-9013-091155f5d528.png)
<img width="439" alt="image" src="https://user-images.githubusercontent.com/35858975/69205773-d3463f00-0baf-11ea-8b12-e8959394fa42.png">


If I remember correctly, the multi columns only get validated on the `patientsOnTreatment` column, but I could be wrong and actually they won't get validated at all.. :( 

I think we'd have to add another column to regimen data in `programSettings` .. `elmisCode` or something which would map it to `patientsOnTreatment`, `patientsOnTreatmenChildren` etc to be able to match them .. maybe something to consider for the regimen data 2.0..?

Reply:
oh. ugh! I was ignoring that scenario. probably should be considering it, as the mSupply changes are being done now